client/src/components/editor/EditorHome.js

```diff
@@ -20,12 +27,65 @@ const EditorHome = () => {
         <>
             <Container className="editor-home-container border">
 
-                <Button variant="primary" block onClick={handleShow}>
-                    <BsFillPlusCircleFill />
+                {/* Add home page editor template Viewer here */}
+
+                <Navbar bg="dark" variant="dark">
+                    <Navbar.Brand href="#home">{header}</Navbar.Brand>
+                    <Nav className="mr-auto">
+                        <Nav.Link href="#home">Home</Nav.Link>
+                        <Nav.Link href="#publications">Publications</Nav.Link>
+                        <Nav.Link href="#awards">Awards</Nav.Link>
+                        <Nav.Link href="#team">Team</Nav.Link>
+                        <Nav.Link href="#contact">Contact</Nav.Link>
```

Comment:
I think we're not using `href` in Link, we're using something like `to=`

Reply:
yep, I agree bootstrap uses to instead.